### DISCUSSION TITLE
How to change to a custom viewer on the connect bentley site?

### DISCUSSION THREAD

**Discussion by imperidal** [iTwin/itwinjs-core#2264] [Q&A] [answered]
Would it be possible to replace the viewer to a custom viewer at the bentley connect page for specific projects? [https://connect.bentley.com/](url)

![image](https://user-images.githubusercontent.com/38965496/133533732-f84cc295-c1f2-41d2-8ed8-75860f640ac1.png)

**Comment by aruniverse** [comment, marked as answer]
If I understand your question correctly, you're asking if its possible to open your own viewer instead of DesignReview in IModelHub/connect.bentley.com. If so, this is not possible and not something we plan to add support for. 

You can open whatever models you have access to in your iTwin Viewer as long as you have the context/project and iModel id. Is there some specific functionality in iModelHub/connect.bentley.com that you're looking to add to your app?

**Reply by imperidal** [reply]
Thanks for the reply.  We like everything in the DesignReview and we were thinking if it's possible to view 2D profile/cross section when clicking on a road geometry for example.

On that note, is there package manager i could pull to get the DesignReview template?  Would love to be able to use that as the starting point for expanding our custom iTwin Viewer.

**Reply by candcconsulting** [reply]
PS : Have you tried using the civil tools cross section
![image](https://user-images.githubusercontent.com/19268085/133963348-c540fa2f-422b-4d55-9eb8-719e182b9712.png)
and this video also highlights some capabilities
https://www.youtube.com/watch?v=0mLoSgyxTxE

**Reply by aruniverse** [reply]
There is no "Design Review" template, as Design Review is a full fledged application. We are looking into providing certain features from Design Review as Extensions to the iTwin Viewer. We are undergoing some changes in iTwin.js 3.0 that will make this process much easier moving forward so stay tuned as new features become available.
